### PR TITLE
feat: re-enable groupBy queries as basis for aggregate view

### PR DESCRIPTION
**Pull request by mswertz**

### CODE REVIEW

File: backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlQuery.java [hunk 1]
```diff
@@ -574,13 +574,9 @@ private Field<?> jsonAggregateSelect(
       if (COUNT_FIELD.equals(field.getColumn())) {
         fields.add(count().as(COUNT_FIELD));
       } else if (GROUPBY_FIELD.equals(field.getColumn())) {
```
Comment (jelmerveen) [inline backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlQuery.java:576]: Should the aggregate not be able to group and count and the same time?

**Comment by jelmerveen** [comment]
When trying a multiple groupBy, the count stays the same.

```
{
  Samples_agg {
    count,
    groupBy {
    	Diseases {
        name
      },
      MaterialTypeDetailed {
        code
      }
  	}
  }
}
```

**Comment by jelmerveen** [comment]
when not using a count, query hangs. Maybe check and return error?

**Comment by jelmerveen** [comment]
Seems also to return a distinct list as a result

**Comment by mswertz** [comment]
[#9370 created group by graqhpql endpoint](https://trello.com/c/yJhCfdeJ/1024-9370-created-group-by-graqhpql-endpoint)

**Review by jelmerveen — APPROVED** [review]

**Pull request merged**